eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1576,19 +1576,6 @@ public function publishVersion(APIVersionInfo $versionInfo, array $translations
             $versionInfo->versionNo
         );
 
-        $fromContent = null;
-        if ($content->contentInfo->currentVersionNo !== $versionInfo->versionNo) {
-            $fromContent = $this->internalLoadContent(
-                $content->contentInfo->id,
-                null,
-                $content->contentInfo->currentVersionNo
-            );
-            // should not occur now, might occur in case of un-publish
-            if (!$fromContent->contentInfo->isPublished()) {
-                $fromContent = null;
-            }
-        }
```

Comment:
Hmm, I might have made a mistake when rebasing #2585. AFAIR it was supposed to be used for `Target\Version` passed as `$targets` to `canUser`... or indeed it's a left-over and it's solved somewhere else. 

@pawbuj However please look at [EZP-30997](https://jira.ez.no/browse/EZP-30997). Maybe this is a root cause of that issue.

See `\eZ\Publish\Core\Repository\ContentService::createContentDraft` for target builder usage. For `publishVersion` I expected to find here `translateToAnyLanguageOf` call.

FYI @konradoboza

Reply:
Indeed it seems to be a dead code but I also have a feeling that intention was to use `$fromContent`  as an argument of `canUser`. Could you please double-check it?

Reply:
https://github.com/ezsystems/ezpublish-kernel/pull/2858 here is the fix to EZP-30997.
I've checked the code  and `publishVersion` no needs `Target\Version`, this is also written here: https://github.com/ezsystems/ezpublish-kernel/pull/2585 . So PR is valid and this pece of code should be removed.